azure-spring-boot-samples/azure-documentdb-spring-boot-sample/README.md

```diff
@@ -33,8 +33,8 @@ java -jar target/azure-ducumentdb-spring-boot-sample-0.0.1-SNAPSHOT.jar
 ```
 ### Known issue
 
-Directly running the sample app from IDE IntelliJ or Eclipse has below security exception if using the *released* starter. The root cause
- is that the release `spring-data-azure-documentdb` jar is code-signed by us. We're working actively to resolve this issue. 
+For `spring-data-documentdb` version before `0.1.4`(excluded), directly running the sample app from IDE IntelliJ or Eclipse has below security exception if using the *released* starter. The root cause
+ is that the release `spring-data-documentdb` jar is code-signed by us.
```

Comment:
no need to start new line

Reply:
root cause is that cert verification issue in spring-data, which will be fixed in spring data.

Reply:
updated.